ui/components/MesheryPerformance/index.js

```diff
@@ -901,7 +901,7 @@ class MesheryPerformanceComponent extends React.Component {
                         <TextField
                           id="contentType"
                           name="contentType"
-                          label="Content Type"
+                          label='Content Type e.g. "application/json"'

```

Comment:
@adithyaakrishna please remove the double quotes, ie
```diff
- label='Content Type e.g. "application/json"'
+ label='Content Type e.g. application/json'
```

The text is already stringified, so the double quotes would be interpreted literally.

Reply:
Okay